### PR TITLE
vswhere: add page

### PR DESCRIPTION
**Pull request by aminya**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 2.8.4 (the latest)

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]

File: pages/windows/vswhere.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+> Locate Visual Studio 2017 and newer installations.
+> More information: <https://github.com/microsoft/vswhere>.
+
+- Find the path of vcvarsall.bat to set environment variables:
+
+`vswhere -products * -latest -prerelease -find "**/VC/Auxiliary/Build/vcvarsall.bat"`
+
+- Find the directory of the x64 MSVC compiler (cl.exe, etc):
+
+`vswhere -products * -latest -prerelease -find **/Hostx64/x64/*`
```
Comment (marchersimon) [inline pages/windows/vswhere.md:12]:
```suggestion
`vswhere -products {{*}} -latest -prerelease -find {{**/Hostx64/x64/*}}`
```
Also, why did you use `" "` in the first example, but not here?

File: pages/windows/vswhere.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Find the directory of the Clang bundled with Visual Studio bundled (clang-cl, clang-tidy, etc):
```
Comment (marchersimon) [inline pages/windows/vswhere.md:14]:
```suggestion
- Find the directory of Clang bundled with Visual Studio bundled (clang-cl, clang-tidy, etc):
```

File: pages/windows/vswhere.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`vswhere -products * -latest -prerelease -find **/Llvm/bin/*`
```
Comment (marchersimon) [inline pages/windows/vswhere.md:16]:
```suggestion
`vswhere -products {{*}} -latest -prerelease -find {{**/Llvm/bin/*}}`
```

File: pages/windows/vswhere.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Find the path of the MSBuild.exe :
```
Comment (marchersimon) [inline pages/windows/vswhere.md:18]:
```suggestion
- Find the path of `MSBuild.exe`:
```

File: pages/windows/vswhere.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`vswhere -products * -latest -prerelease -find MSBuild/**/Bin/MSBuild.exe`
```
Comment (marchersimon) [inline pages/windows/vswhere.md:20]:
```suggestion
`vswhere -products {{*}} -latest -prerelease -find {{MSBuild/**/Bin/MSBuild.exe}}`
```

File: pages/windows/vswhere.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# vswhere
+
+> Locate Visual Studio 2017 and newer installations.
+> More information: <https://github.com/microsoft/vswhere>.
+
+- Find the path of vcvarsall.bat to set environment variables:
```
Comment (marchersimon) [inline pages/windows/vswhere.md:6]: I'm not sure what this `to set the environment variables` part does here. Could you please explain this a bit?

File: pages/windows/vswhere.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+`vswhere -products * -latest -prerelease -find "**/VC/Auxiliary/Build/vcvarsall.bat"`
```
Comment (marchersimon) [inline pages/windows/vswhere.md:8]:
```suggestion
`vswhere -products {{*}} -latest -prerelease -find "{{**/VC/Auxiliary/Build/vcvarsall.bat}}"`
```

File: pages/windows/vswhere.md (hunk 1 above)
Comment (aminya) [reply pages/windows/vswhere.md:12]: These are not variables that you want to add `{{}}` around them.

File: pages/windows/vswhere.md (hunk 6 above)
Comment (aminya) [reply pages/windows/vswhere.md:6]: `vsvarsall` is used to set environment variables. See #7439

**Comment by TEPPYSITH** [comment]
Good

**Comment by aminya** [comment]
Any answers to my comment? This is just a simple documentation pull request.

**Review by CleanMachine1 — APPROVED** [review]

**Comment by CleanMachine1** [comment]
Sorry for the delay

**Pull request merged**